code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1057,6 +1058,25 @@
 	REMOVE_TRAIT(M, TRAIT_STABLEHEART, type)
 	REMOVE_TRAIT(M, TRAIT_STABLELIVER, type)
 	..()
+/datum/reagent/medicine/penthrite
+	// Same as corazone. but slightly better
+	name = "Penthrite"
+	description = "An explosive compound used to stabilize heart conditions. May interfere with stomach acid!"
+	color = "#F5F5F5"
+	self_consuming = TRUE
+
+/datum/reagent/medicine/penthrite/on_mob_metabolize(mob/living/M)
+	..()
+	if(prob(10))
+		M.adjustOrganLoss(ORGAN_SLOT_STOMACH,-REM)
+	M.dizziness += 1
+	M.confused += 1
+	ADD_TRAIT(M, TRAIT_STABLEHEART, type)
```

Comment:
this can just be added on add, rather than constantly trying to add it per tick

Reply:
still an issue

Reply:
you never fixed this.

Reply:
i really dont know what to do with this. every single other  chem does this in such a way